backend/spec/models/question_type_spec.rb

```diff
@@ -1,13 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe QuestionType, type: :model do
-  describe 'uniqueness' do
-    subject { create(:question_type) }
+  subject { create(:question_type) }
 
+  describe 'uniqueness' do
     it { is_expected.to validate_uniqueness_of(:name).case_insensitive }
+  end
+
+  describe 'relationships' do
+    it { is_expected.to have_many(:questions).dependent(:destroy) }
+  end
+
+  describe '#destroy' do
+    describe 'When there are questions for question type' do
+      let!(:question_type) { create(:question).question_type }
+
+      it 'does not destroy' do
+        question_type.destroy
```

Comment:
I think this line should go to a `before`

Reply:
True!